bats/tests/helpers/vm.bash

```diff
@@ -17,6 +17,21 @@ pkill_by_path() {
     fi
 }
 
+windows_cleanup() {
+    # Fixed in WSL 0.64.0, however below is a workaround for windows10
+    # https://github.com/microsoft/WSL/releases/tag/0.64.0
+    export WSL_UTF8=1
+    WSLENV="${WSLENV}:WSL_UTF8"
```

Comment:
Why is this necessary? I suspect it should have been removed when we removed grepping the output of `wsl -l -v` for distro names,